app/src/views/CountryNsOverviewContextAndStructure/NationalSocietyLocalUnits/LocalUnitsFormModal/LocalUnitsForm/schema.ts

```diff
@@ -164,21 +166,87 @@ const schema: LocalUnitsFormSchema = {
                                 speciality: {
                                     validations: [lengthSmallerThanCondition(200)],
                                 },
-                                maximum_capacity: {},
-                                number_of_isolation_rooms: {},
+                                maximum_capacity: {
+                                    validations: [
+                                        integerCondition,
+                                        greaterThanOrEqualToCondition(0),
```

Comment:
we could use `positiveIntegerCondition` from `#utils/form`. Let's do this for all the following as well.